prow/plugins/trigger/trigger.go

```diff
@@ -38,6 +38,33 @@ const (
 	PluginName = "trigger"
 )
 
+// untrustedReason represents a combination (by ORing the appropriate consts) of reasons
+// why a user is not trusted by TrustedUser. It is used to generate messaging for users.
+type untrustedReason int
+
+const (
+	notMember untrustedReason = 1 << iota
+	notCollaborator
+	notSecondaryMember
+)
+
+// untrustedReasonString constructs a string explaining the reason for a user's denail of trust
+// from untrustedReason as described above.
+func untrustedReasonString(reasons untrustedReason) string {
```

Comment:
optinal nit: looking at the function name maybe a better fit would be to implement stringer on the type instead.

`func (u untrustedReason) String() string`

Reply:
Done.